pvlib/clearsky.py

```diff
@@ -782,147 +1034,13 @@ def rmse(alpha):
 
     if return_components:
         components = OrderedDict()
-        components['mean_diff_flag'] = c1
-        components['max_diff_flag'] = c2
-        components['line_length_flag'] = c3
-        components['slope_nstd_flag'] = c4
-        components['slope_max_flag'] = c5
-        components['mean_nan_flag'] = c6
+        components['mean_diff'] = c1
+        components['max_diff'] = c2
+        components['line_length'] = c3
+        components['slope_nstd'] = c4
+        components['slope_max'] = c5
+        components['mean_nan'] = c6
```

Comment:
This may have been unintentional. In the current PR, only the Boolean series for each criteria are returned. In v0.8, the Boolean series are returned in the '_flag' columns, and the values for each criteria are returned in the columns named for each criteria e.g. 'line_length'

I'm happy to put it back as in v0.8, if that's agreed.

Reply:
-1 on the API change for 0.8